data/ship builders jobs.txt

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2019 by Peter van der Meer
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Escort Hauler prototype to formal dinner"
```

Comment:
Since this is part of a sequence of missions, it's better to just abbreviate this as something like `mission "Hauler Prototype 1"`, and put `Escort Hauler prototype to formal dinner` in a `name` token below instead.